deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -2550,6 +2550,127 @@ status:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  labels:
+    longhorn-manager: ""
+  name: snapshots.longhorn.io
```

Comment:
Also need to add `snapshots` `uninstall.yaml`

Reply:
Thank you. Fixed